### PR TITLE
Upgrades Robottelo Image upstream build action

### PR DESCRIPTION
**Pull request by jyejare**
Support to build the upgrade upstream image to be used by pipelines:
    - Upgrade scenario 
    - Upgrade All Tiers
    - FM

### CODE REVIEW

File: .github/workflows/merge_to_master.yaml [hunk 1]
```diff
@@ -0,0 +1,82 @@
+name: update_upgrade_image
+
+on:
+  push:
+    branches:
+      - master
```
Comment (jameerpathan111) [inline .github/workflows/merge_to_master.yaml:6]: @jyejare do we want this github action to run only when we push changes to master branch and not others?
This means we will only be creating/updating `upgrade:latest` image and not any other `upgrade:x.y.z` ones.

Comment (jyejare) [reply .github/workflows/merge_to_master.yaml:6]: Yes ! Cause we are building the containers for robottelo/satellite-upgrade only from master branch.

**Comment by jyejare** [comment]
@mshriver Need your quick and merge here .

File: .github/workflows/merge_to_master.yaml (hunk 1 above)
Comment (mshriver) [reply .github/workflows/merge_to_master.yaml:6]: We just updated this in robottelo too, thanks for raising it @jameerpathan111. Easy to update later if we do want to start building z-stream branches for this repo.

**Review by mshriver — APPROVED** [review]

**Pull request merged**